public/stylesheets/style.less

```diff
@@ -42,14 +42,15 @@ div.card {
     display: flex;
     flex-direction: column;
     justify-content: center;
+    width: 100%;
```

Comment:
Why did you added this code?

Reply:
To make the container content take all the remaining space